lib/voice/VoiceConnection.js

```diff
@@ -585,13 +596,36 @@ class VoiceConnection extends EventEmitter {
             if(msg[1] !== 0x78) { // unknown payload type, ignore
                 return;
             }
+            const rtpVersion = msg[0] >> 6;
+            if(rtpVersion !== 2) { // Check if RTP version is 2
+                this.emit("warn", "Recieved packet with an invalid RTP version: " + rtpVersion);
+                return;
+            }
 
+            const hasExtension = !!(msg[0] & 0b10000);
+            const hasPadding = !!((msg[0] >> 5) & 1);
```

Comment:
nit: keep this consistent with `hasExtension`
```suggestion
            const hasPadding = !!(msg[0] & 0b100000);
```